ports/samd/boards/GENERIC_SAMD51P20/board.json

```diff
@@ -0,0 +1,15 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "USB"
+    ],
+    "mcu": "samd51",
+    "product": "Generic SAM51P20",
+    "variants": {
+        "WLAN": "WLAN with SSL support"
```

Comment:
Does this variant exist?

Reply:
Not yet, it's a different PR. And maybe never.